net.go

```diff
@@ -82,6 +82,12 @@ type ping struct {
 	// the intended recipient. This is to protect again an agent
 	// restart with a new name.
 	Node string
+
+	// YouShouldRefute is a courtesy flag that's sent during a probe to
+	// tell the node being probed that it's currently in the suspect or dead
+	// state from the perspective of the prober so it can start right away
+	// trying to refute that.
+	YouShouldRefute bool
```

Comment:
Also I wonder if it makes sense to include the incarnation number of the suspect. e.g. if I "suspect" at 7 but the alive for 10 is already out, then its fine and can be no-oped.